packages/lodestar/src/api/impl/node/index.ts

```diff
@@ -51,13 +51,34 @@ export function getNodeApi(opts: IApiOptions, {network, sync}: Pick<ApiModules,
     },
 
     async getPeerCount() {
-      // TODO: Implement
+      let disconnected = 0;
```

Comment:
So disconnected means all peers that have ever connected to us and are no longer connected? That sounds like a memory leak to track lol

Reply:
Yup. Lighthouse has a `network_globals` object that tracks this. So when a peer is disconnected, it is tracked, as can be seen here [here](https://github.com/sigp/lighthouse/blob/9bc9527998581deb5fd8674052bb2083b640bbbd/beacon_node/lighthouse_network/src/peer_manager/mod.rs#L714) which is then read and used to return the peer count as can be seen [here](https://github.com/sigp/lighthouse/blob/9bc9527998581deb5fd8674052bb2083b640bbbd/beacon_node/http_api/src/lib.rs#L1811) only that the `network_globals` is not persistent.

Reply:
It's persisted to disk that's how it doesn't leak memory. I would leave a `// TODO: Implement disconnect count with on-disk persistence`

Reply:
I assumed it is not being persisted as the values don't persist after each restarts. Will dig later to see how that is implemented... 

>  I would leave a // TODO: Implement disconnect count with on-disk persistence

Ok. Will do.

Reply:
> Will dig later to see how that is implemented...

I would ignore completely, tracking disconnected peers is an extremely unnecessary feature in my opinion haha